node/routes.ts

```diff
@@ -79,6 +74,7 @@ const handleAdyenWebhook = async (ctx: Context) => {
       true
     )
 
+    // eslint-disable-next-line no-console
```

Comment:
Rather than disabling the eslint rule, I would prefer that the console statements be removed